src/common/browser-adapters/webextension-browser-adapter.ts

```diff
@@ -206,7 +216,9 @@ export abstract class WebExtensionBrowserAdapter
     }
 
     public getUrl(urlPart: string): string {
-        return chrome.extension.getURL(urlPart);
+        return typeof chrome.extension?.getURL === 'function'
```

Comment:
Is falling back to `chrome.extension.getURL` necessary here? I think `browser.extension.getURL` is documented as working in all cases we support, regardless of manifest version